### ISSUE TITLE
Documentation and IDD Issues - September-October 2015

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5241]
- [x] Input Output Reference, four pipe fan coil,

```
1.30.3.26 Filed: Supply Air Fan Operating Mode Schedule Name
This input field is the name of a schedule that controls fan operation. Schedule Name values of 0 denote cycling fan operation (fan cycles with cooling coil). Schedule values greater than 0 denote constant fan operation (fan runs continually regardless of coil
operation). The fan operating mode defaults to cycling fan operation if this field is left blank. This input field is currently used with MultiSpeedFan capacity control
method only.
```
1.  Filed --> Field
2.  @Nigusse Does the fan really cycle only with cooling coil?  Or should this say "fan cycles with heating or cooling coil"?

**Comment by mjwitte** [comment]
- [x] Helpdesk ticket 10788
  Input Output Reference for Chiller:Electric and Chiller:Electric:EIR needs to state clearly that heat recovery only works with Condenser Type = WaterCooled.  Probably should add this to the three Heat Recovery field descriptions.  Also IDD notes.

Also, in Chiller:Electric, Field: Design Heat Recovery Water Flow Rate, there is a typo "Enegineering Manual".

**Comment by Nigusse** [comment]
@mjwitte It should say "fan cycles with heating or cooling coil".

**Comment by mjwitte** [comment]
- [x] Helpdesk ticket 10842 People object, various output variables for the ASHRAE 55 and CEN 15251 thermal comfort models say "a value of -1 means not applicable."  It would be useful to spell out when the models are applicable in the People "Thermal Comfort Model Type" field description.  

Engineering Reference for ASHRAE 55 says "If, using either method, To is less than 10°(C) or greater than 33.5°(C), the model is not applicable." where To is monthly mean outdoor air dry-bulb temperature (°C).

Engineering Reference for CEN 15251 is vague: "If Trm is not within the specified domain, the model is not applicable."  where Trm = weighted running average of the previous 7 daily average outdoor air
temperatures.  The specified domain appears to be 15-30C based on the figure in that section, but this should be confirmed.  
- [x] A clear statement of the temperature range should be added to the Engineering Ref for this one as well as stating it in the I/O Ref.
- [x] Also, according to the user, output variable "Zone Thermal Comfort CEN 15251 Adaptive Model Temperature" sometimes reports -1, but the description for this variable does not include the phrase "and a value of -1 means not applicable."  Check to see if this is true.  Also, check "Zone Thermal Comfort ASHRAE 55 Adaptive Model Temperature"

_MJW 07 Dec 2015_
In the source code ThermalComfort.cc these outputs are applicable when shown below, otherwise set to -1
For Zone Thermal Comfort ASHRAE 55 Adaptive Model . . . Acceptability Status
`if ( runningAverageASH >= 10.0 && runningAverageASH <= 33.5 ) {`
`if ( numOccupants > 0 ) {`
For Zone Thermal Comfort ASHRAE 55 Adaptive Model Temperature [C]
`if ( runningAverageASH >= 10.0 && runningAverageASH <= 33.5 ) {`

For Zone Thermal Comfort CEN 15251 Adaptive Model Category . . . Status
`if ( runningAverageCEN >= 10.0 && runningAverageCEN <= 30.0 ) {`
`if ( numOccupants > 0 ) {`

For Zone Thermal Comfort CEN 15251 Adaptive Model Temperature [C]
`if ( runningAverageCEN >= 10.0 && runningAverageCEN <= 30.0 ) {`

**Comment by mjwitte** [comment]
- [x] IDD for Sizing:System Type of Load to Size On, \notes don't reflect the new Total option

```
  A2, \field Type of Load to Size On
      \note Specifies the basis for sizing the system supply air flow rate
      \note Sensible and VentilationRequirement are the only available options
      \note Sensible uses the zone design air flow rates
      \note VentilationRequirement uses the system ventilation requirement
      \type choice
      \key Sensible
      \key Total
      \key VentilationRequirement
      \default Sensible
```

**Comment by mjwitte** [comment]
- [x] Chiller:Electric:EIR, plr curve is limited to Quadratric, user file has a cubic which runs fine, suggest letting this be UnivariateCurves.  Other chillers as well?  (ref. helpdesk ticket 10861 but can reproduce with a very simple example file change)

```
  A4 , \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Name
       \note Electric Input Ratio (EIR) as a function of Part Load Ratio (PLR)
       \note EIR = 1/COP
       \required-field
       \type object-list
       \object-list QuadraticCurves
       \object-list UniVariateTables
```

**Comment by mjwitte** [comment]
- [x] Ref Helpdesk Ticket 10868.  Illuminance maps inputs for number of x/y grid points should have a minimum of 1.  A fraction can cause a crash.
  `\minimum 1`

```
Output:IlluminanceMap,
  N4 , \field Number of X Grid Points
       \note Maximum number of total grid points must be <= 2500 (X*Y)
       \type integer
       \minimum> 0
       \default 2
  N7 ; \field Number of Y Grid Points
       \note Maximum number of total grid points must be <= 2500 (X*Y)
       \type integer
       \minimum> 0
       \default 2
```

**Comment by mjwitte** [comment]
@wfbuhl Sometime during v8.3.0, in Sizing:System, field Type of Load to Size On, the key choice "Total" was uncommented and added to the I/O ref as a valid choice, but no explanation was added (in IDD or I/O Ref).  Looking at the code, the getinput takes it but I don't see Sensible or Total making any difference in the actual sizing code.  Is this a real choice or not?

**Comment by wfbuhl** [comment]
1.17.10.2 Field: Type of Load to Size On
The user specified type of load on which to size the central system. The choices are  Sensible, Total and VentilationRequirement. Sensible means that the central
system supply air flow rate will be determined by combining the zone design air flow rates, which have been calculated to meet the zone sensible loads from the
design days. VentilationRequirement means that the central system supply air flow rate will be determined by the system ventilation requirement. In addition
Sensible tells the program to size the central cooling coil using entering air flow rate and air conditions at the sensible load peak; Total indicates that the program
should size the central cooling coil at the air flow rate and conditions at the total load peak. The central heating coil is always sized at the conditions at the peak
sensible heating load.

I think it is explained in the above IO Ref; though the word "Total" should be italicized in the IO Ref doc. The issue as I now vaguely remember it is that the central cooling coil  was always being sized on the flow rate at the sensible peak; now it can be sized on the flow rate at the total peak. This was part of the coil vs space peak issue. In the code you'll see it used in function GetCoilDesFlowT. So it is a real choice. I would just comment that the notes in the IDD are now incorrect and looks like it was never altered to reflect "total" becoming an active key value.

**Comment by mjwitte** [comment]
@wfbuhl Duh - I guess I missed the last sentence because it's not italicized.  I'll fix that.  And now I see it gets transferred to .CoolingPeakLoadType which then gets used in GetCoilDesFlow.  Thanks for pointing me in the right direction.

**Comment by mjwitte** [comment]
@wfbuhl And just to clarify - the system supply air flow rate will be the same with Sensible and Total?  Just the coil sizing will change between these two?

**Comment by wfbuhl** [comment]
Yes

**Comment by mjwitte** [comment]
Closed via #5352
